Helpers/RequestHelper.cs

```diff
@@ -120,11 +130,11 @@ internal static async Task<MessageWithStatusCode> PostFileRequest(PyrusClient cl
 
 		internal static async Task<MessageWithStatusCode> GetFileRequest(PyrusClient client, string url, string token)
 		{
-			using (var httpClient = client.ClientSettings.NewHttpClient(FileRequestTimeout))
+			using (var httpClient = client.ClientSettings.NewHttpClient())
 			{
 				httpClient.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", token);
 				httpClient.DefaultRequestHeaders.Add("User-Agent", UserAgent);
-
+				httpClient.Timeout = FileRequestTimeout;
```

Comment:
We are overriding Timeout and UserAgent. Shouldn't we consent with developer decision?